### DISCUSSION TITLE
Parsing XML strings question

### DISCUSSION THREAD

**Discussion by ironyman** [PowerShell/PowerShell#15987] [Q&A] [answered]
We can create an XmlDocument like so
```
> [xml]"<a></a>"
a
-
```
but the following fails
```
> $a = "<a></a>"
> [xml]$a
InvalidArgument: Cannot convert value "System.Xml.XmlDocument[]" to type "System.Xml.XmlDocument". Error: "The specified node cannot be inserted as the valid child of this node, because the specified node is the wrong type."
```

Why does this happen?

**Comment by vexx32** [comment, marked as answer]
That doesn't occur for me... what version of pwsh are you running on?

**Reply by ironyman** [reply]
v7.1.3

Hmm I'm not able to repro this anymore. Sorry maybe I did something weird :\

**Reply by ironyman** [reply]
I would mark as answer on this but the button isn't working :|
